common/source/docs/common-mamba405-mk2.rst

```diff
@@ -0,0 +1,172 @@
+================================
+Mamba F405 MK2 Flight Controller
+================================
+
+.. image:: ../../../images/mamba_f405mk2_board.jpg
+     :target: ../_images/mamba_f405mk2_board.jpg
+
+The Mamba F405 MK2 is a flight controller produced by
+`Diatone <https://www.diatone.us>`__.
+
+
+Features
+========
+
+-  STM32F405RGT6 microcontroller
+-  MPU6000 IMU
+-  AT7456E OSD
+-  4 UARTs
+-  4 PWM outputs
+
+Where to buy
+============
+
+Available from multiple retailers and directly from the vendor at `diatone.us <https://www.diatone.us/products/mamba-f405-flight-controller-mk2>`__
+
+
+Pinout
+======
+
+.. figure:: ../../../images/mamba_f405mk2_pinout.png
```

Comment:
we need a new pinout/wiring diagram for ArduPilot....UART3 is GPS by default in the hwdef...none is shown in the BF diagram...also we would need to show an external I2C BARO on the I2C bus along with an optional compass...